client/src/components/Properties/Property.tsx

```diff
@@ -425,16 +425,16 @@ const Property = ({
     }, []);
 
     useEffect(() => {
-        const loadDependencyOptions = optionsDataSource?.loadOptionsDependsOn?.reduce(
+        const loadOptionsDependency = optionsDataSource?.loadOptionsDependsOn?.reduce(
```

Comment:
🚨 ovo je ilegalno 👮 . Definiraš varijablu koja ima isto ime kao i state, nešto od toga mora imati drugačije ime